_includes/types.html

```diff
@@ -15,13 +15,13 @@ <h6 class="m-0 d-flex align-items-center">
     {% endif %}
   </h6>
   <div class="btn-group">
-    <a href="https://download.opensuse.org{{type.primary_link}}" class="btn btn-{{ distro.bg-color }} text-{{ distro.fg-color }}">{{ locale.download }}</a>
+    <a href="https://mirrorcache.opensuse.org{{type.primary_link}}?PEDANTIC=1" class="btn btn-{{ distro.bg-color }} text-{{ distro.fg-color }}">{{ locale.download }}</a>
     <button type="button" class="btn btn-{{ distro.bg-color }} text-{{ distro.fg-color }} dropdown-toggle dropdown-toggle-split" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
       <span class="sr-only">Toggle Dropdown</span>
     </button>
     <div class="dropdown-menu dropdown-menu-right">
       {% for link in type.links %}
-      <a href="https://download.opensuse.org{{link.url}}" class="dropdown-item">{{ locale[link.name] }}</a>
+      <a href="https://mirrorcache.opensuse.org{{link.url}}" class="dropdown-item">{{ locale[link.name] }}</a>
```

Comment:
Should this also have PEDANTIC set?

Reply:
No, these are metalink request - they have a list of mirrors in reply and PEDANTIC is not supported there. (I am not sure if it is good idea to wait response untill all the mirrors are checked up).

Reply:
Makes sense